src/controllers/expensesController.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const { usersService } = require('../services/users');
+const { expensesService } = require('../services/expenses');
+
+const getAll = (req, res) => {
+  const { userId, category, from, to } = req.query;
+  const expenses = expensesService.getAll(userId, category, from, to);
+
+  res.send(expenses);
+};
+
+const findOne = (req, res) => {
+  const { expenseId } = req.params;
+
+  const foundExpense = expensesService.findExpenseById(Number(expenseId));
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundExpense);
+};
+const addOne = (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+  const foundUser = usersService.findUserById(userId);
+
+  if (!foundUser) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  if (typeof userId !== 'number' || typeof spentAt !== 'string'
```

Comment:
```suggestion
  if (!userId || typeof spentAt !== 'string'
```
do you really need this type of checks or you can go with just an '!'?